### ISSUE TITLE
How it's related to tensorflow quantization methods?

### ISSUE THREAD

**Issue by mrgloom** [tensorflow/compression#13]
How it's related to tensorflow built-in quantization methods? i.e. post-training quantization and quantization-aware training.

**Comment by jonycgn** [comment]
Hello mrgloom,

the TF built-in quantization is usually applied to all activations in an artificial neural network, with the goal to speed up computations without losing prediction accuracy.

Our work here quantizes only the bottleneck, and it does so under an entropy constraint (i.e. it minimizes the average length of the compressed representation). This is a quite different objective.

Having said that, we have also explored quantizing the entire architecture, essentially combining the two goals. For that, see our most recent paper [here](https://openreview.net/forum?id=S1zz2i0cY7).

I hope that answers your question! If you have further comments, please post to our Google group [here](https://groups.google.com/forum/#!forum/tensorflow-compression). Issues are more for reporting bugs – I'm closing this one.

Best,
Johannes
